<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>312</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>494</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <item row="3" column="3">
     <widget class="QPushButton" name="ButtonMult">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: #FF8C00;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #FF9F00;
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Button6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: grey;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #A9A9B0;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="ButtonChangeSign">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: grey;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #A9A9B0;
}</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Button4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: grey;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #A9A9B0;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="ButtonAdd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: #FF8C00;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #FF9F00;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="Button0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: grey;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #A9A9B0;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="ButtonDiv">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: #FF8C00;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #FF9F00;
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="ButtonSub">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: #FF8C00;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #FF9F00;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Button5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: grey;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #A9A9B0;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="Button7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: grey;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #A9A9B0;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="Button8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: grey;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #A9A9B0;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="ButtonMMinus">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: #FF8C00;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #FF9F00;
}</string>
      </property>
      <property name="text">
       <string>M-</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="ButtonAC">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: grey;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #A9A9B0;
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="ButtonMPlus">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: #FF8C00;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #FF9F00;
}</string>
      </property>
      <property name="text">
       <string>M+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="Button3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: grey;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #A9A9B0;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="Button1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: grey;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #A9A9B0;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="Button2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: grey;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #A9A9B0;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="Button9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: grey;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #A9A9B0;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="ButtonM">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: #FF8C00;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #FF9F00;
}</string>
      </property>
      <property name="text">
       <string>M</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="ButtonEq">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: black;
	background-color: #FF8C00;
	border: none;
	padding: none;
}

QPushButton:pressed {
	background-color: #A9A9A9;
}

QPushButton:hover {
	background-color: #FF9F00;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2" colspan="5">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color:grey;
	border: 1px solid gray;
	padding: 5px;
	color: white;
	margin-bottom: 5px;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
